opencog/relex/Dockerfile

```diff
@@ -0,0 +1,98 @@
+#
+# Docker file that builds RelEx and starts the RelEx server.
+#
```

Comment:
I manually synced up both the relex and this dockerfile to be the same.  I also like the new start/stop scripts here, which relex doesn't have ...